RequestManagementSystem/Client/Request.py

```diff
@@ -550,22 +553,34 @@ def optimize( self ):
           removeRepList.append( ( op2.TargetSE, op2 ) )
           del self.__operations__[i]
           del self.__operations__[i]
-          continue
+          print 'Found sequence starting at %d' % i
+          optFound = True
+      if not optFound:
```

Comment:
Perfectly correct... It fixes the problem but creates an even bigger one! 
Conclusion: don't implement a fix while listening to a conference!